charts/nx-cloud/templates/nx-cloud-nx-api-deployment.yaml

```
@@ -86,6 +86,10 @@ spec:
             - name: ACTIVEMQ_ADDRESS
               value: tcp://nx-cloud-messagequeue:61616
           {{- end }}
+          {{- if .Values.nxCloudWorkflows.enabled }}
+            - name: NX_CLOUD_CI_SERVER_ADDRESS
+              value: http://nx-cloud-workflow-controller-service:9000
```

I guess this value shouldn't be hardcoded here?

```suggestion
              value: {{ .Values.nxCloudWorkflows.ciServerAddress }}:9000
```